src/book/02-system/01-foundations/04-nuclear.mdx

```diff
@@ -186,6 +186,20 @@ NBS files are NUClear Binary Stream files. To make it easier to serialise stream
 
 All values within this format are little endian.
 
+An `nbz` or `nbs.gz` file is a compressed `nbs` file using gzip.
```

Comment:
```suggestion
An `.nbz` or `.nbs.gz` file is a compressed `nbs` file using gzip.
```

Reply:
Do we want to put a dot here? Above, for normal `nbs` files, we don't have a dot.